src/Field/Data/Table/TableRowData.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Storyblok\Field\Data\Table;
+
+final class TableRowData
```

Comment:
Do we need that VO?

Reply:
Potentially. A full table row (within the `tbody`) contains more meta data such as field type and the corresponding uid for the row.

If we don't want it, I wouldn't mind removing the row VO and just work with raw `TableColumnData` arrays.

Reply:
I would remove it for now. We can always expand later.